Cargo.toml

```diff
@@ -329,6 +329,7 @@ self_cell = "1.0.4"
 selinux = "0.4.4"
 signal-hook = "0.3.17"
 smallvec = { version = "1.13.2", features = ["union"] }
+sysinfo = "0.29.10"
```

Comment:
Thinking more about it, I am a bit reluctant to add a new crate just for this.
It can probably be done using libc or nix, no ?